framework/db/BaseActiveRecord.php

```diff
@@ -104,6 +104,15 @@ public static function findOne($condition)
 
     /**
      * @inheritdoc
+     * @return static|null ActiveRecord instance matching the condition, or `null` if nothing matches.
+     */
+    public static function findFirst($condition)
```

Comment:
Better to add ```$sort``` parameter, cause no guarantees that ```limit 1``` will always return the same record.